apps/availability-monitor/src/review/index.ts

```diff
@@ -96,16 +101,18 @@ const checks = {
         peers.connected.length >= minAmountOfPeers,
         peers.connected.length,
       ];
-    }
+    },
+    CHECK_TIMEOUT
   ),
   hoprdOpenOutgoingChannels: createCheck<Channel[], [HoprSDK, number]>(
     "hoprd-open-outgoing-channels",
     async function checkHoprdOpenChannels(sdk, minChannels) {
-      const channels = await sdk.api.channels.getChannels();
+      const channels = await sdk.api.channels.getChannels({});
```

Comment:
curious. how does it work with empty payload?

Reply:
its simply a typing change in the library as I updated it, behaves the same